cckn/programmers/JadenCase 문자열 만들기/app.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+  return s
+    .split(' ')
+    .map((str) => {
+      if (str.length === 0) return str
+      const first = str[0].match(/[a-z]/i) ? str[0].toUpperCase() : str[0]
+      return `${first}${str.slice(1).toLowerCase()}`
```

Comment:
```suggestion
      return `${str[0].toUpperCase()}${str.slice(1).toLowerCase()}`
```

str[0]이 알파벳인지 검사할 필요 없이 그냥 toUpperCase만 적용했어도 됐겠다.